src/org/zaproxy/zap/extension/ascanrulesAlpha/resources/Messages_es_ES.properties

```diff
@@ -77,13 +77,13 @@ ascanalpha.ldapinjection.knownimplementations=activedirectory\:openldap\:389dire
+ascanalpha.ldapinjection.openldap.errormessages=Operaciones de error\: Protocolo de error\: L\u00edmite de tiempo excedido\: L\u00edmite de tama\u00f1o excedido\: Comparar Falso\: Comparar Verdadero\: m\u00e9todo de autenticaci\u00f3n no admitido\: Fuerte (ER) de autenticaci\u00f3n requerido\: l\u00edmite administrativo excedi\u00f3\: extensi\u00f3n cr\u00edtica no est\u00e1 disponible\: La confidencialidad requerida\: bind SASL en curso\: n tales atributos\: Indefinido tipo de atributo\: coincidente inapropiado\: restricci\u00f3n de violaci\u00f3n\: existe tipo o el valor\: la sintaxis no v\u00e1lida\: no existe el objeto\: un problema Alias\: sintaxis inv\u00e1lida DN\: Alias \u200b\u200bdereferencing problema\: la autenticaci\u00f3n inapropiado\: credenciales no v\u00e1lidas\: el acceso insuficiente\: El servidor est\u00e1 ocupado\: servidor no est\u00e1 disponible\: El servidor no est\u00e1 dispuesto a realizar\: bucle detectado\: Naming violaci\u00f3n\: violaci\u00f3n clase de objeto\: Operaci\u00f3n no permitida en la no-hoja\: Operaci\u00f3n no permitida en RDN\: ya existe\: no se puede modificar la clase de objeto\: aprovechamiento afecte a m\u00faltiples DSA\: vista de lista virtual de error\: Otros (por ejemplo, la aplicaci\u00f3n espec\u00edfica) error\: Los resultados parciales y la remisi\u00f3n recibieron\: la entrada es una hoja\: resultados demasiado grande\: no Operaci\u00f3n para cancelar\: demasiado tarde para cancelar\: no puede Canc EL\: Error de aserci\u00f3n\: Error de aserci\u00f3n (X)\: con proxy autorizaci\u00f3n es denegada\: Autorizaci\u00f3n proxy fallo (X)\: Contenido de sincronizaci\u00f3n de actualizaci\u00f3n requerido\: Contenido de sincronizaci\u00f3n de actualizaci\u00f3n requerida (X)\: No Operaci\u00f3n (X)\: Recursos LCUP de existencias\: Violaci\u00f3n LCUP de seguridad\: LCUP Los datos no v\u00e1lido\: no compatible LCUP Esquema\: LCUP Recargar requerido\: TXN especificar bien\: TXN ID no es v\u00e1lido\: no se puede contactar con el servidor LDAP\: error local\: error de codificaci\u00f3n\: La decodificaci\u00f3n de error\: Tiempo de espera agotado\: m\u00e9todo de autenticaci\u00f3n Desconocido\: filtro de b\u00fasqueda malo\: el usuario cancel\u00f3 la operaci\u00f3n\: Bad par\u00e1metro a una rutina de LDAP\: Fuera de la memoria\: Conectar error\: no se admite\: control no encontrado\: no hay resultados devueltos\: M\u00e1s resultados para volver\: Cliente Loop\: L\u00edmite de Referencia excedido\: API error desconocido\: error desconocido (extensi\u00f3n)\: Desconocido ( extensi\u00f3n privada) error\: error desconocido\n\u00a0
```

Comment:
I'm unconvinced that this was pulled from the library source as the comment outlines.

Reply:
Yeah, does no look like it. I'll open a PR to move these to other file for the time being.

Reply:
Done in #1326.